packages/dds/map/src/directory.ts

```diff
@@ -428,24 +430,31 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
         super(id, runtime, attributes);
         this.localValueMaker = new LocalValueMaker(this.serializer);
         this.setMessageHandlers();
+        // Mirror the containedValueChanged op on the SharedDirectory
+        this.root.on(
+            "containedValueChanged",
+            (changed: IValueChanged, local: boolean, op: ISequencedDocumentMessage) => {
+                this.emit("containedValueChanged", changed, local, op);
```

Comment:
>containedValueChanged [](start = 27, length = 21)

as i call out on the interfaces file, this event isn't defined on ISharedDirectory, so i don't think it is part of the typing here

Reply:
Fixed in latest